components/Workspaces/TrackedRepoWizard/PasteReposStep.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import Button from "components/atoms/Button/button";
+
+interface PasteReposStepProps {
+  onBulkAddRepos: (repos: string[]) => void;
+}
+
+export const PasteReposStep = ({ onBulkAddRepos }: PasteReposStepProps) => {
+  const [pastedInput, setPastedInput] = useState("");
+
+  const parseInput = () => {
+    // split each line into a trimmed string
+    const repos = pastedInput.split("\n").map((line) => line.trim());
+    onBulkAddRepos(repos);
+  };
+
+  return (
+    <div className="flex flex-col gap-4 h-96 max-h-96">
+      <p className="text-red-500">NOTE: repository names are case sensitive</p>
+      <textarea
+        value={pastedInput}
+        onChange={(e) => setPastedInput(e.target.value)}
+        placeholder="Paste repositories here line by line (e.g. open-sauced/app)"
+        className="p-4 border rounded-xl h-full"
```

Comment:
When the textarea gets focus, it uses the default blue outline of the browser. There are custom focus styles already set in the application. It's a little more involved to add it to the global styles for now, so in the interim, you can add the focus styles to the textarea in this component.

```
outline-none focus-visible:ring focus-visible:border-orange-500 focus-visible:ring-orange-100
```